### DISCUSSION TITLE
Need help in calculating total time spent in hours on enrolled and consumed courses.

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#186] [Issues]
Hello team
Hope you are doing well

Background -
We have updated few screens for sunbird-Ed and trying to add few more details

Issue Details -
I wanted to know weather we have any API that provide us actual spent time on the courses by specific user. And in which database table we can find this data. We don't want to calculate it based on start date and end as it can not be precise

@komalm, @manishakapadnis, @AmiableAnil

**Comment by ManojNathIC** [comment]
Hello team, 

Just a quick follow-up on the above regarding the API for actual spent time on courses by specific users in sunbird-Ed. 

Any guidance on this would be greatly appreciated.

Thank you!
